### DISCUSSION TITLE
Validation of product S1 RTC gamma0 raster

### DISCUSSION THREAD

**Discussion by alexisahedo** [johntruckenbrodt/pyroSAR#238] [Q&A] [answered]
Dear @johntruckenbrodt,

Using pyroSAR I processed S1 images in the search of creating a radar mosaic, for such, the preprocessing of the S1 images to RTC gamma0 is a big first step, but I have no mean to validate and compare the results of the processing. 
**Could you please recommend me any tool, strategy, or dataset to validate and review the rasters resulted?** 

I would like to produce quality products for the sake of the project I am working with and have the means to prove it.

As always, I appreciate your help.

Saludos.
Alexis A.

**Comment by johntruckenbrodt** [comment, marked as answer]
Good question, @alexisahedo. Imho, there is no clear consensus in the community yet on how to do it. I have done a fairly large study some years ago but there is clearly more to be done. Here's the paper and the corresponding source code:
https://doi.org/10.3390/data4030093
https://github.com/johntruckenbrodt/S1_ARD

Here's a very interesting recent development in RTC methodology that might also give you some ideas:
https://doi.org/10.1109/TGRS.2022.3147472

edit:
This topic is of course very relevant for the development of the [ESA S1-NRB](https://sentinel.esa.int/web/sentinel/sentinel-1-ard-normalised-radar-backscatter-nrb-product) product in which I am involved in.

**Reply by alexisahedo** [reply]
As always, I really appreciate your help, I’ll read the docs referenced with detail. Also, I’ll follow closely your other projects, thanks for your good work.

Saludos.

Alexis Ahedo.
